modules/Forum/pages/forum/view_topic.php

```diff
@@ -392,12 +392,9 @@
     }
 }
 
-if ($user->isLoggedIn()) {
```

Comment:
This safe to remove?

Reply:
Yes, this is leftover from the quote revamp PR from a while ago!